paperslicer/grobid/figures.py

```
@@ -0,0 +1,121 @@
+        if facs:
+            first = facs.split()[0]
+            if first in zones:
+                page, bbox = zones[first]
+        # fallback: sometimes coords are inline attributes
+        if bbox is None and fig.get("coords"):
+            try:
+                # try formats: "x1,y1,x2,y2" or similar
+                parts = [float(x) for x in fig.get("coords").replace(",", " ").split()[:4]]
+                if len(parts) == 4:
+                    bbox = parts
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)